cg/services/order_validation_service/workflows/balsamic/models/sample.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import Field
+from cg.constants.constants import GenomeVersion
+from cg.models.orders.sample_base import NAME_PATTERN, ControlEnum, SexEnum, StatusEnum
+from cg.services.order_validation_service.constants import ElutionBuffer, TissueBlockEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class BalsamicSample(Sample):
+    age_at_sampling: float | None = None
+    capture_kit: str | None = None
+    comment: str | None = None
+    concentration_ng_ul: float | None = None
+    control: ControlEnum | None = None
+    elution_buffer: ElutionBuffer
```

Comment:
Should this not be optional?

Reply:
Not according to the docs https://atlas.scilifelab.se/infrastructure/orders/workflows/balsamic_qc_requirements/

Reply:
The docs are wrong 😅